### ISSUE TITLE
Listeners descriptor mismatch 

### ISSUE THREAD

**Issue by iliyan-velichkov** [eclipse/dirigible#2967]
**Describe the bug**
Listener descriptor in the UI is different from the descriptor which should be defined in a project file.
UI:
![image](https://github.com/eclipse/dirigible/assets/5058839/5f3c95b4-2e5b-400c-a501-51e144408a5a)
project listener:
![image](https://github.com/eclipse/dirigible/assets/5058839/ae2031fa-3b86-422e-a857-db83fd0423b9)

**Expected behavior**
Will be better if we unify the scheme.

**Comment by delchev** [comment]
Yes, it should be `kind` in the descriptor now. The `type` is the former one used before Spring migration (pre 8.0)
